devfile/devfile-web
# Use correct syntax as specified in the API

The [tests](https://github.com/devfile/api/blob/v2.2.0/pkg/validation/variables/test-fixtures/all/devfile-good.yaml) uses a `{{ ... }}` syntax, whereas this documentation says to use `{ ... }` syntax.  The double-braces are correct (e.g. odo uses double braces).

## What does this PR do / why we need it

The documented syntax doesn't work, and I wasted a bit of time figuring out what I was doing wrong.

## Which issue(s) does this PR fix

None

## PR acceptance criteria



## How to test changes / Special notes to the reviewer

I guess the {{ .. }} syntax may break some build scripts, so it's wise to check that these {{ ... }} make it through the pipelines and out on the other end.
